integration-test/common/conftest.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import os
+import tempfile
+import shutil
+import subprocess
+
+RUN_SCRIPT_FNAME = "run.sh"
+OUTPUT_DIR = "output_dir"
+SWEEP_GENERATOR = "generate_design_sweeps.py"
+
+def is_master(config):
+  """Return true we're on a master node."""
+  return not hasattr(config, 'workerinput')
+
+def pytest_addoption(parser):
+  parser.addoption("--sweep-file", action="store", help="Sweep file")
+
+def pytest_configure(config):
+  """Generate sweeps using the sweep generator.
+
+  Only the master node performs the sweep generation.
+  """
+  if is_master(config):
+    cwd = os.getcwd()
```

Comment:
Does it matter if cwd is not in /tmp? For example, if cwd is somewhere in the repo, I don't think we want to be polluting that - the user could cancel the test in the middle and leave a bunch of temp files in the repo.

Reply:
For `tmp_output_dir`, it doesn't matter. For `tmp_sweep_file`, it does because of the relative path of machsuite_constants.xe. I remove the cwd constraint for `tmp_output_dir`, where most of the temporary sweeps are generated.